### PR TITLE
feat: Updating reporter panel's default width to match new designs

### PR DESCRIPTION
**Pull request by tbiethman**
- Closes [UNIFY-1168](https://cypress-io.atlassian.net/browse/UNIFY-1168)

### Additional details

New designs put the default reporter panel width at 450px, up from 320px.

### How has the user experience changed?

Old:
Screen Shot 2022-02-16 at 11 10 31 PM

New:
Screen Shot 2022-02-16 at 11 08 39 PM

### PR Tasks

- [X] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/app/src/runner/ResizablePanels.cy.tsx [hunk 1]
```diff
@@ -163,9 +163,10 @@ describe('<ResizablePanels />', { viewportWidth: 1500, defaultCommandTimeout: 40
 
       cy.contains('panel2').should('not.be.visible')
       assertWidth('panel1', defaultPanel1Width)
-      dragHandleToClientX('panel1', 1000)
-      assertWidth('panel1', 1000)
-      assertWidth('panel3', 500)
+      dragHandleToClientX('panel1', 950)
+      dragHandleToClientX('panel1', 955)
```
Comment (tbiethman) [inline packages/app/src/runner/ResizablePanels.cy.tsx:167]: @marktnoonan I was surprised that this test started failing due to panel 1 no longer resizing to 1000px. But it turns out we still account for panel 2's width when deciding panel 1's maximum size, even when panel 2 is hidden. It doesn't look like we disable panel 2 anywhere but the tests, but I'm curious if this behavior is intentional.

Comment (tbiethman) [reply packages/app/src/runner/ResizablePanels.cy.tsx:167]: Agreed on the additional tests, I'll log something to track this. We'll want get a bunch of workflows snapshotted once our Percy workflows are more stable.

As far as the `showPanel2` prop goes, do we even want to continue supporting it? Or do we anticipate a need in the near future?

**Review by marktnoonan — DISMISSED** [review]
LGTM. From a UX perspective I have some reservations about how this will work on smaller windows, but I'll bring that up separately, doesn't block this change.

File: packages/driver/cypress/e2e/commands/actions/trigger.cy.js [hunk 2]
```diff
@@ -782,11 +782,15 @@ describe('src/cy/commands/actions/trigger', () => {
 
     describe('position argument', () => {
```
Comment (tbiethman) [inline packages/driver/cypress/e2e/commands/actions/trigger.cy.js:783]: With the change to the default width, the AUT is scaled differently as the third panel now has less room. Firefox behavior varies in how it the AUT scrolls at different scales when the buttons are injected into the DOM for these tests, and that impacts the assertions we are making regarding client coordinates. You can see where this was a problem for the existing size at lines 828/846.

I updated the other tests that were being made that are impacted by this variability. I also left comments and tried to make the meaning of the numbers in the calculations a little more clear.

**Review by davidmunechika — APPROVED** [review]

**Review by marktnoonan — APPROVED** [review]

**Pull request merged**